files/jcl/ZWESECUR.jcl

```diff
@@ -520,32 +557,36 @@ $$
 
 /* DEFINE ZOWE SERVER PERMISIONS ................................... */
 
-/* required updates:                                                 */
-/* - change "fac_owning_acid" to the acid that owns IBMFAC           */
-
-/* permit Zowe main server to use XMEM, cross memory server          */
-  TSS ADD(fac_owning_acid) IBMFAC(ZWES.IS)
+/* permit Zowe main server to use XMEM cross memory server           */
+  TSS ADD(&FACACID.) IBMFAC(ZWES.IS)
   TSS WHOHAS IBMFAC(ZWES.IS)
   TSS PERMIT(&ZOWEUSER.) IBMFAC(ZWES.IS) ACCESS(READ)
 
+/* permit Zowe main server to create a user's security environment   */
+/* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes    */
+/*            z/OS UNIX switch to z/OS UNIX level security. This is  */
+/*            more secure, but it can impact operation of existing   */
+/*            applications. Test this thoroughly before activating   */
+/*            it on a production system.                             */
+  TSS ADD(&FACACID.) IBMFAC(BPX.)
+  TSS WHOHAS IBMFAC(BPX.SERVER)
+  TSS PER(&ZOWEUSER.) IBMFAC(BPX.SERVER) ACC(UPDATE)
+
 /** comment out to not use SUPERUSER.FILESYS, see JCL comments       */
 /** permit Zowe main server to write persistent data                 */
-    TSS ADD(fac_owning_acid) UNIXPRIV(SUPERUSE)
+    TSS ADD(&FACACID.) UNIXPRIV(SUPERUSE)
     TSS WHOHAS IBMFAC(SUPERUSER.FILESYS)
     TSS PER(&ZOWEUSER.) UNIXPRIV(SUPERUSER.FILESYS) ACCESS(CONTROL)
 
 /* DEFINE XMEM SERVER PERMISIONS ................................... */
 
-/* required updates:                                                 */
-/* - change "fac_owning_acid" to the acid that owns IBMFAC           */
-
 /* permit XMEM to create a user's security environment               */
 /* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes    */
-/*            z/OS UNIX switch to z/OS UNIX level security, which is */
+/*            z/OS UNIX switch to z/OS UNIX level security. This is  */
 /*            more secure, but it can impact operation of existing   */
 /*            applications. Test this thoroughly before activating   */
 /*            it on a production system.                             */
-  TSS ADD(fac_owning_acid) IBMFAC(BPX.)
+  TSS ADD(&FACACID.) IBMFAC(BPX.)
   TSS WHOHAS IBMFAC(BPX.DAEMON)
   TSS PER(&XMEMUSER.) IBMFAC(BPX.DAEMON) ACC(UPDATE)
   TSS WHOHAS IBMFAC(BPX.SERVER)
```

Comment:
BPX.SERVER and BPX.DAEMON access for &XMEMUSER is not required.

Reply:
as before, ZSS does need it, it also switches security context